### ISSUE TITLE
Add more metrics

### ISSUE THREAD

**Issue by yangtcai** [yangtcai/Ensembl-Repeat-Identification#48]
Currently, The mAP has been chosen as the metric, however, In the recent training result, I found the value of mAP is always 0, I think it is good to add more metrics, such as precision, and recall. It can help us figure out why the mAP is always 0.

**Comment by williamstark01** [comment]
Good idea, go ahead with it! For mAP in particular, it may be worth debugging it line by line and by running experiments with small datasets.
